test/functional/pruning.py

```diff
@@ -331,6 +322,14 @@ def wallet_test(self):
     def run_test(self):
         self.log.info("Warning! This test requires 4GB of disk space and takes over 30 mins (up to 2 hours)")
         self.log.info("Mining a big blockchain of 995 blocks")
+
+        # Determine default relay fee
+        self.relayfee = self.nodes[0].getnetworkinfo()["relayfee"]
+
+        # Cache for utxos, as the listunspent may take a long time later in the test
+        self.utxo_cache_0 = []
+        self.utxo_cache_1 = []
```

Comment:
Because I'd really prefer `__init__()` to only set up the network topology. (eventually I think we should try to move away from individual test cases overriding the `__init__()` method entirely).